docs/playbook/ingesting-data/011-ingesting-api-data-2.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Ingesting data from an API to the Data Platform 2
+description: "Ingesting API data into the Data Platform using an AWS Lambda function 2"
+layout: playbook_js
+tags: [playbook]
+---
+# Ingesting Data from an API into the Data Platform - 2
```

Comment:
Is this " - 2" intentional?

Reply:
For now, I want to keep the previous documentation there until its safe to remove. Looking at the old documentation it looks like theres a section on alarming now added, so I should add it to mine soon